_spinnaker/account_management_api.md

```diff
@@ -0,0 +1,186 @@
+    data:
+      mgmtapi.yml: |
+        storage:
+          github:
+            owner: {owner}
+            repo: {repo}
+            password: {redacted}
+            file: clouddriver-remote.yml
+            baseUrl: {baseUrl for Github Enterprise, if appropriate}
+    kind: ConfigMap
+    metadata:
+      name: acct-mgmt-api-config
+    ---
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      labels:
+        app: acct-mgmt-api
+      name: acct-mgmt-api
+    spec:
+      replicas: 1
+      selector:
+        matchLabels:
+          app: acct-mgmt-api
+      template:
+        metadata:
+          labels:
+            app: acct-mgmt-api
+        spec:
+          containers:
+            - image: armory/acct-mgmt-api:0.0.1-7fb4dfd-dev0000
+              name: api
+              ports:
+                - containerPort: 3000
+              volumeMounts:
+                - mountPath: /opt/spinnaker/config
+                  name: config
+          volumes:
+            - configMap:
+                name: acct-mgmt-api-config
+              name: config
+    ---
+    apiVersion: v1
+    kind: Service
+    metadata:
+      labels:
+        app: acct-mgmt-api
+      name: acct-mgmt-api
+    spec:
+      ports:
+        - port: 3000
+          protocol: TCP
+          targetPort: 3000
+      selector:
+        app: acct-mgmt-api
+      type: ClusterIP
+    
+
+
+## Configuration
+
+Configuring the Account Management API similar to other Spinnaker services. For Github integration, add the config (shown above). This enables the Github integration and uses the supplied credentials when making changes.
+
+*There is a sample configuration in the manifests above.*
+
+
+    # mgmtapi.yml
+    storage:
+      github:
+        owner: armory-io
+        repo: spinnaker-remote-config
+        password: {redacted}
+        file: clouddriver-remote.yml
+
+When using the Github integration, you need to configure Clouddriver to use Github as well. Note, the `remote` profile corresponds to the `clouddriver-remote` file that the API manages. To enable remote configuration, add the snippet below to `~/.hal/{DEPLOYMENT_NAME}/profiles/spinnakerconfig.yml`.
+
+    spring:
+      profiles:
+        include: git,remote
+      cloud:
+        config:
+          server:
+            git:
+              uri: https://github.com/armory-io/spinnaker-remote-config 
+              password: {redacted}
+              username: doogie
+
+More details on configuring Spinnaker for remote configuration can be found [here.](https://www.spinnaker.io/setup/configuration/)
+
+## Usage
+
+The Account Management API works by hooking into the same backends as Spring’s Cloud Config - the core of Dynamic Accounts within Spinnaker. For example, if you are using Github as a backend for storing your account configuration, you can use the same backend with the Account Management API. Changes are reflected into Clouddriver as the Management API makes changes to the backing data.
```

Comment:
Nit: 

- Github -> GitHub
- (passive vs active voice) When the Management API changes the backing data, it replicates the change to Clouddriver.
?